### PR TITLE
Added recent Erlang versions

### PR DESCRIPTION
**Pull request by OnorioCatenacci**
I added the information for two more recent versions of Erlang that are
available.  18.2 and 18.2.1

### CODE REVIEW

File: erlang.csv [hunk 1]
```diff
@@ -1,7 +1,9 @@
 version_otp,version_erts,url_win32,url_win64
+18.2.1,7.2.1,http://erlang.org/download/otp_win32_18.2.1.exe,http://erlang.org/download/otp_win64_18.2.1.exe
+18.2,7.2,http://erlang.org/download/otp_win32_18.2.exe,http://erlang.org/download/otp_win64_18.2.exe
 18.1,7.1,http://www.erlang.org/download/otp_win32_18.1.exe,http://www.erlang.org/download/otp_win64_18.1.exe
 18.0,7.0,http://www.erlang.org/download/otp_win32_18.0.exe,http://www.erlang.org/download/otp_win64_18.0.exe
 17.5,6.4,http://www.erlang.org/download/otp_win32_17.5.exe,http://www.erlang.org/download/otp_win64_17.5.exe
 17.4,6.3,http://www.erlang.org/download/otp_win32_17.4.exe,http://www.erlang.org/download/otp_win64_17.4.exe
 17.3,6.2,http://www.erlang.org/download/otp_win32_17.3.exe,http://www.erlang.org/download/otp_win64_17.3.exe
-17,6.0,http://www.erlang.org/download/otp_win32_17.0.exe,http://www.erlang.org/download/otp_win64_17.0.exe
+17,6.0,http://www.erlang.org/download/otp_win32_17.0.exe,http://www.erlang.org/download/otp_win64_17.0.exe
```
Comment (josevalim) [inline erlang.csv]: Please don't remove the new line at the end!

Comment (OnorioCatenacci) [inline erlang.csv]: Oops--my bad.  I'll fix it.  Sorry Jose!

Comment (OnorioCatenacci) [inline erlang.csv]: Fixed Jose. Sorry for the additional annoyance! :)

On Wed, Jan 27, 2016 at 11:38 AM, José Valim <notifications@github.com>
wrote:

> In erlang.csv
> <https://github.com/elixir-lang/elixir-lang.github.com/pull/671#discussion_r51010589>
> :
>
> >  18.1,7.1,http://www.erlang.org/download/otp_win32_18.1.exe,http://www.erlang.org/download/otp_win64_18.1.exe
> >  18.0,7.0,http://www.erlang.org/download/otp_win32_18.0.exe,http://www.erlang.org/download/otp_win64_18.0.exe
> >  17.5,6.4,http://www.erlang.org/download/otp_win32_17.5.exe,http://www.erlang.org/download/otp_win64_17.5.exe
> >  17.4,6.3,http://www.erlang.org/download/otp_win32_17.4.exe,http://www.erlang.org/download/otp_win64_17.4.exe
> >  17.3,6.2,http://www.erlang.org/download/otp_win32_17.3.exe,http://www.erlang.org/download/otp_win64_17.3.exe
> > -17,6.0,http://www.erlang.org/download/otp_win32_17.0.exe,http://www.erlang.org/download/otp_win64_17.0.exe
> > +17,6.0,http://www.erlang.org/download/otp_win32_17.0.exe,http://www.erlang.org/download/otp_win64_17.0.exe
>
> Please don't remove the new line at the end!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir-lang.github.com/pull/671/files#r51010589>
> .
>



-- 
Onorio Catenacci

http://onor.io
http://www.google.com/+OnorioCatenacci

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**